src/main/scala/org/allenai/deep_qa/experiments/datasets/Science.scala

```diff
@@ -135,18 +261,18 @@ object ScienceDatasets {
 object ScienceFiles {
   def makeMultipleTrueFalseQuestionAnswerFile(questionFile: String, outputFile: String): JValue = {
     ("sentence producer type" -> "question interpreter") ~
-    ("create sentence indices" -> true) ~
-    ("question file" -> questionFile) ~
-    ("output file" -> outputFile) ~
-    ("interpreter" -> ("type" -> "append answer"))
+      ("create sentence indices" -> true) ~
```

Comment:
Yes, this is definitely more readable.  My editor doesn't do this by default, so that's why it was how it was.  I'll have to see if I can fix that...